### PR TITLE
feat(demo): fix web vitals and context generation

### PR DESCRIPTION
**Pull request by scefali**
This PR does the following:
1. Generates random `browser` and `os` fields for an event
2. Changes some logic for durations of front end transactions to be configurable
3. Bases web vital times off the length of the transaction duration
4. Tweaks some data gen settings

### CODE REVIEW

**Review by wmak — APPROVED** [review]
Suggestion to add a mobile OS too, otherwise LGTM

File: src/sentry/demo/data/os_list.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  {
+    "name": "Mac OS X",
+    "version": "10.14.6",
+    "type": "os"
+  },
+  {
+    "name": "Windows",
+    "version": "10",
+    "type": "os"
+  }
```
[the remark this answers is not in the document]
Comment (scefali) [reply src/sentry/demo/data/os_list.json:16]: @wmak hmmm I think we want this scenario to be a web app so I am going to hold off on adding mobile (but at some point we are planning on adding a mobile scenario).

[the remark this answers is not in the document]
Comment (wmak) [reply src/sentry/demo/data/os_list.json:16]: 🤷 minor suggestion so feel free to ignore, but web apps will have mobile traffic too won't they?

[the remark this answers is not in the document]
Comment (scefali) [reply src/sentry/demo/data/os_list.json:16]: @wmak good point

**Pull request merged**